zebra-consensus/src/block/check.rs

```diff
@@ -15,28 +15,46 @@ use crate::{error::*, parameters::SLOW_START_INTERVAL};
 
 use super::subsidy;
 
-/// Returns `Ok(())` if there is exactly one coinbase transaction in `Block`,
-/// and that coinbase transaction is the first transaction in the block.
+/// Checks if there is exactly one coinbase transaction in `Block`,
+/// and if that coinbase transaction is the first transaction in the block.
+/// Returns the coinbase transaction is successful.
 ///
 /// "The first (and only the first) transaction in a block is a coinbase
 /// transaction, which collects and spends any miner subsidy and transaction
 /// fees paid by transactions included in this block." [§3.10][3.10]
 ///
```

Comment:
It's section 3.11 now, and this is what it says:
> A transaction that has a single transparent input with a null prevout field, is called a coinbase transaction. Every block has a single coinbase transaction as the first transaction in the block. The purpose of this coinbase transaction is to collect and spend any miner subsidy, and transaction fees paid by other transactions included in the block.

Reply:
Thanks for catching this, fixed in 3659a7c68